lib/archethic/utils.ex

```diff
@@ -60,27 +60,39 @@ defmodule Archethic.Utils do
   """
   @spec configurable_children(
           list(
-            process ::
-              atom()
-              | {process :: atom(), args :: list()}
-              | {process :: atom(), args :: list(), opts :: list()}
+            (process ::
+               any()
+               | {process :: atom(), args :: list()}
+               | {process :: atom(), args :: list(), opts :: list()})
+            | %{id: atom(), start: {process :: atom(), fx :: atom(), args :: list(any())}}
```

Comment:
What is the difference with Supervisor.child_spec ?

Reply:
Cache lru and lru disk are using child specs. And we need to refactor that part with behavior impl. for now we just accept specs as given .

Reply:
Yes but this is define in the `Supervisor.child_spec()` typespec